apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -22,7 +22,7 @@ namespace Google.Cloud.Diagnostics.Common
     /// <summary>
     /// The location error events will be sent to.
     /// </summary>
-    public enum ReportEventsToLocation
+    public enum EventTarget
```

Comment:
I'm not *hugely* happy with it to be honest. It's a more detailed EventTarget, it feels like...  I think I'd personally call the class EventTarget and the enum EventTargetKind or EventTargetType.

How much are end-users going to use this? The degree of bike-shedding that makes sense will depend on that :)